auth/basics/android.md

```diff
@@ -147,6 +204,17 @@ skygear.getAuth().signupAninymously(new AuthResponseHandler() {
     }
 });
 ```
+```kotlin
+skygear.auth.signupAnonymously(object : AuthResponseHandler() {
+  override fun onAuthSuccess(user: Record) {
+    Log.i("Skygear Signup", "onAuthSuccess: Got token: ${user.getAccessToken()}")
```

Comment:
I don't think `getAccessToken` exists.

Reply:
Yeh me neither.. I was simply copying `getAccessToken` from the Java code but I  couldn't find such function in class `Record` which is `user`'s class. Does this mean such function doesn't exist, hence the Java version won't work as well?

Reply:
`getAccessToken` is originally included in the Java code. Such function cannot be found on either Java nor Kotlin side, think it might existed in the past. I did find a similar function

```java
final Container skygear = Container.defaultContainer(this);
skygear.getAuth().getCurrentAccessToken();
```
in `Skygear.Container`. 

There are 14 usages of the non-existent function `getAccessToken`, 7 on each side. How would you like them to be replaced? Cheers.

Reply:
@elly0t  Yes,  I think you can stick to `skygear.getAuth().getCurrentAccessToken()`. Sorry the Java version is misleading. `AccessControl` here means ACL, therefore it is a completely separate concept.